packages/core/data/mm/world/overworld.yml

```diff
@@ -720,7 +720,7 @@
   exits:
     "Road to Southern Swamp": "true"
     "Tourist Information": "true"
-    "Swamp Back": "event(BOAT_RIDE) || event(CLEAN_SWAMP) || has_mask_zora || (has(MASK_DEKU) && (has_arrows || can_hookshot_short))"
+    "Swamp Back": "event(BOAT_RIDE) || event(CLEAN_SWAMP) || has_mask_zora || (has(MASK_DEKU) && (has_arrows || can_hookshot_short) || fd_anywhere)"
```

Comment:
This would likely be its own trick, but i'd have to look back at MMR logic. It would be a Big Octo skip.